tests/profiles.py

```diff
@@ -1,8 +1,6 @@
-#!/usr/bin/env python
```

Comment:
some of the following formatting changes should probably be on their own commit, consider changing